pkg/domain/infra/abi/parse/parse.go

```diff
@@ -78,6 +78,16 @@ func VolumeOptions(opts map[string]string) ([]libpod.VolumeCreateOption, error)
 					libpodOptions = append(libpodOptions, libpod.WithVolumeDisableQuota())
 					// set option "NOQUOTA": "true"
 					volumeOptions["NOQUOTA"] = "true"
+				case "timeout":
+					if len(splitO) != 2 {
+						return nil, errors.Wrapf(define.ErrInvalidArg, "gid option must provide a GID")
+					}
+					intTimeout, err := strconv.Atoi(splitO[1])
+					if err != nil {
+						return nil, errors.Wrapf(err, "cannot convert Timeout %s to an integer", splitO[1])
+					}
+					logrus.Debugf("Removing timeout from options and ading WithTimeout for Timeout %d", intTimeout)
```

Comment:
in case you breeze by it the first time like I did
```suggestion
					logrus.Debugf("Removing timeout from options and adding WithTimeout for Timeout %d", intTimeout)
```